.travis.yml

```diff
@@ -69,6 +69,10 @@ jobs:
       if: type = push AND tag =~ /^v\d+\.\d+(\.\d+)?(-\S*)?$/ AND env(DOCKER_USERNAME) IS present
       script: make release
       name: Release Artifacts Linux
+    - stage: Deploy
```

Comment:
oops I missed this, it should go after the windows release.

Reply:
we were prob both thinking everything important was already done